app/models/host_storage.rb

```diff
@@ -10,7 +10,7 @@ class HostStorage < ApplicationRecord
 
   class << self
     def writable_accessible
-      writable.accessible
+      writable.accessible.includes(:storages).where.not(:storage => maintenance_mode)
```

Comment:
I think you mean `joins(:storages)`

`includes` tends to refer to local ruby objects.

If `joins` doesn’t work, then I guess we need `references(:storages)`

Reply:
bleagh, i hate it. thanks.